metadata-ingestion/setup.py

```diff
@@ -532,8 +539,10 @@ def get_long_description():
         "oracle = datahub.ingestion.source.sql.oracle:OracleSource",
         "postgres = datahub.ingestion.source.sql.postgres:PostgresSource",
         "redash = datahub.ingestion.source.redash:RedashSource",
-        "redshift = datahub.ingestion.source.sql.redshift:RedshiftSource",
-        "redshift-usage = datahub.ingestion.source.usage.redshift_usage:RedshiftUsageSource",
```

Comment:
breaking change. Please see if we can avoid it. Also, need to update [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md) file